Helper/Cache/CacheHelper.php

```diff
@@ -53,4 +55,20 @@ public function save(CacheItem $item, array $data)
 
         $this->cache->save($item);
     }
+
+    public static function setCacheHeaders(Request $request, Response $response): Response
+    {
+        if (is_string($response->getContent())) {
+            $response->setCache([
+                'etag' => \md5($response->getContent()),
```

Comment:
don't like the hardcoded md5, maybe we should add the hash algo parameter to the commonBundle?

Reply:
Based on that article I defined sha1 as default etag_hash_algo a0ad9062551337ee00f364f3db4aa75cb693dfc9